src/themes/light.scss

```diff
@@ -18,6 +18,9 @@
   --wxc-modal-border: 1px solid #{$wxc-black-alpha-11};
   --wxc-modal-box-shadow: 0 1.5rem 3rem #{$wxc-black-alpha-20}, 0 0 0.0625rem #{$wxc-black-alpha-12};
 
+  --wxc-spinner-circle-color: #{$wxc-light-gray};
```

Comment:
use `$wxc-gray-30` instead, it's the closest color, and let's ask @lalli-flores what to do with non standard color codes.